.github/workflows/build-website.yml

```diff
@@ -11,64 +11,92 @@ jobs:
   build:
     name: website
     runs-on: ubuntu-latest
-    # TODO(strager): Use an image based on
-    # quick-lint-js-web.
-    container: ghcr.io/quick-lint/quick-lint-js-github-builder:v3
+    container: ghcr.io/quick-lint/quick-lint-js-website-builder:v1
     env:
       QLJS_COLLECT_COPYRIGHT_NO_WARNINGS: 1
+      EM_VERSION: 2.0.4
+      EM_CACHE_FOLDER: 'emsdk-cache'
+
     steps:
-      - uses: quick-lint/setup-emsdk@v7
-        with:
-          version: 2.0.4
       - name: checkout
         uses: actions/checkout@v2
 
+      - name: setup emsdk cache
+        id: cache-system-libraries
+        uses: actions/cache@v2
+        with:
+          path: ${{env.EM_CACHE_FOLDER}}
+          key: ${{env.EM_VERSION}}-${{ runner.os }}
+
+      - uses: quick-lint/setup-emsdk@v7
+        with:
+          version: ${{env.EM_VERSION}}
+          actions-cache-folder: ${{env.EM_CACHE_FOLDER}}
```

Comment:
We should really put the emscripten installation in the Docker container instead.

Reply:
size of emscripten is about 1.11 GB. see [screenshot](https://puu.sh/Jd5rf/7794b36cde.png)